publish.gradle

```diff
@@ -0,0 +1,127 @@
+    ext["signing.secretKeyRingFile"] = project.hasProperty('SigningSecretKeyRingFile') ? "$SigningSecretKeyRingFile" : ''
+    ext["ossrhUsername"] = project.hasProperty('OSSRHUsername') ? "$OSSRHUsername" : ''
+    ext["ossrhPassword"] = project.hasProperty('OSSRHPassword') ? "$OSSRHPassword" : ''
+    ext["sonatypeStagingProfileId"] = project.hasProperty('SonatypeStagingProfileID') ? "$SonatypeStagingProfileID" : ''
+}
+
+
+project.ext.bintrayPublishFunc = { bintrayContext, aarName ->
+    bintrayContext.user = project.hasProperty('bintrayUser') ? "$bintrayUser" : ""
+    bintrayContext.key = project.hasProperty('bintrayKey') ? "$bintrayKey" : ""
+    bintrayContext.publications = ['FluentUI']
+    bintrayContext.publish = true
+    bintrayContext.override = true
+    bintrayContext.pkg {
+        repo = 'generic'
+        name = aarName
+        version {
+            name = android.defaultConfig.versionName
+            vcsTag = android.defaultConfig.versionName
+        }
+    }
+}
+
+project.ext.publishingFunc = { artifactIdName ->
+    publishing {
+        repositories {
+            maven {
+                url 'https://pkgs.dev.azure.com/microsoftdesign/fluentui-native/_packaging/fluentui-android/maven/v1'
+                credentials {
+                    username = project.hasProperty("mavenUserName") ? "$mavenUserName" : ""
+                    password = project.hasProperty("mavenPassword") ? "$mavenPassword" : ""
+                }
+            }
+            maven {
+                name = "sonatype"
+                url = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
+                credentials {
+                    username ossrhUsername
+                    password ossrhPassword
+                }
+            }
+        }
+        publications {
+            FluentUI(MavenPublication) {
+                groupId 'com.microsoft.fluentui'
+                artifactId artifactIdName
+                version = android.defaultConfig.versionName
+                artifact(sourceJar)
+                artifact(bundleReleaseAar)
+                pom {
+                    name = artifactIdName
+                    description = 'Fluent UI Android library'
+                    url = 'https://github.com/microsoft/fluentui-android'
+                    licenses {
+                        license {
+                            name = 'MIT License'
+                            url = 'https://github.com/microsoft/fluentui-android/blob/master/LICENSE'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'fluent-ui-android'
+                            name = 'Microsoft Inc.'
+                            email = 'fluentuinativeowners@microsoft.com'
+                        }
+                    }
+                    scm {
```

Comment:
should we declare Urls and trivial properties in gradlewrapper.properties, would be more readable ?